app/src/main/java/com/twilio/voice/quickstart/VoiceActivity.java

```diff
@@ -297,36 +290,40 @@ public void onDestroy() {
 
     private void handleIncomingCallIntent(Intent intent) {
```

Comment:
What is the behavior of the application on Android O or higher when the user just clicks the notification body and not "Accept" or "Reject"?